Gemfile.lock

```diff
@@ -3,103 +3,84 @@ GEM
   specs:
     addressable (2.6.0)
       public_suffix (>= 2.0.2, < 4.0)
-    capybara (2.7.1)
```

Comment:
Oof we need a `Gemfile.lock -diff` line in our `.gitattributes` to not show this lock file for PRs.

Reply:
Done.

Reply:
Though, it looks like github doesn't respect that. Local git did.